readme.md

```diff
@@ -9,27 +9,6 @@ With NodeJS installed, just run NPM:
 
 > npm install can-compile -g
 
-## Command line
```

Comment:
Why is this moved down? I would like people to read about the command line usage before talking about build tools.